### ISSUE TITLE
ImageDownloadButton

### ISSUE THREAD

**Issue by halimahLDN** [Greater-London-Authority/ldn-viz-tools#764]
See #741 
- [x] City Intelligence logo doesn't have alt text:
![image](https://github.com/user-attachments/assets/b7fa3ab0-26cd-4cf7-b4f3-f1eb59ef1481)
